### PR TITLE
29 improve tester documentation

### PR DESCRIPTION
**Pull request by gregLibert**

### CODE REVIEW

File: tester/check_replacer.js [hunk 1]
```diff
@@ -178,26 +196,26 @@ describe(${args[0]}, ():i32 => {
...
-      expr += `return TestResult.Success;\n});\n`;
+      expr += `\n  return TestResult.Success;\n});\n`;
 
       // magic function define at parent level that will:
       // - remove the code used here from the initial file content
       // - create a new file with the generated tests
       this.addUpdate({
         begin: node.range.start,
-        end: node.range.end+2, // +1 to include the trailing semicolon and new line (;\n)
+        end: node.range.end + 2, // +1 to include the trailing semicolon and new line (;\n)
```
Comment (thomas-senechal) [inline tester/check_replacer.js:218]: Isn't it supposed to be `+2` in that comment ?

```suggestion
        end: node.range.end + 2, // +2 to include the trailing semicolon and new line (;\n)
```

**Review by Thykof — COMMENTED** [review]
there are lots of check failure github annotations,  are they relevant?

File: tester/check_replacer.js (hunk 1 above)
Comment (gregLibert) [reply tester/check_replacer.js]: fixes

**Review by Thykof — APPROVED** [review]
LGTM 🤷 but we will need to refactor and remove eslint-disable ^^

**Pull request merged**